ui-components/src/tables/StandardTable.tsx

```diff
@@ -36,9 +36,18 @@ export interface StandardTableProps {
 export const StandardTable = (props: StandardTableProps) => {
   const { headers, data, cellClassName } = props
 
+  // Translation strings are optionally passed in with arguments with format `key,argumentName:argumentValue`
+  const getTranslatedHeader = (headerKey: string) => {
+    if (headerKey.indexOf(",") >= 0) {
+      const translation = headerKey.split(",")
+      const argument = translation[1].split(":")
```

Comment:
Seems like a good approach for the arguments syntax. Could we make it more flexible so multiple pairs supported by commas are allowed?

`t.key.path,val1:Value 1,val2:Value2`